README.md

```diff
@@ -1 +1,33 @@
-Bryson's first Spring Boot app (kind of)
\ No newline at end of file
+Configuring Service as Outgoing Webhook in Slack
+===============================================
+Navigation
+----------
+1. From your [slack home page][1], while logged in, choose _Customize Slack_ option in the drop-down menu at the top left corner of the page. A new __Customize Your Team__ window should be opened.
+2. Click on the _Configure Apps_ option in the menu to the left. A new window with a list of apps should be opened.
+3. Choose _Custom Integrations_ on the menu to the left.
+4. In the list of integrations, choose _Outgoing Webhooks_. A list of current E-gineering Custom integrations will be listed.
+5. If a Chuck Norris Configuration isn't already present, click the _Add Configuration button_, and follow the set-up instructions below. If one does exist, then just click on the edit button to the right of the current Chuck Norris configuration.
+
+The settings for the service should be as follows:
+==================================================
+__Channel__: Any
+
+__Trigger Words__: chucknorris (this will allow upper- and lowercase anywhere in the word, i.e. chuCkNorris will be accepted)
+
+__URL__: https://rest-joke.herokuapp.com/joke (or wherever the service is being deployed)
```

Comment:
You'll want to avoid including real URLs like this since it will be open to the public and we don't really want to advertise our server.  I would just say something like "URL of your service endpoint" that will accept the POST from Slack.